stack.yaml

```diff
@@ -132,5 +139,7 @@ flags:
 # See https://github.com/ndmitchell/weeder/issues/53
 ghc-options: {"$locals": -ddump-to-file -ddump-hi}
 
+allow-newer: true
```

Comment:
It would be preferable not to use this. We should be able to avoid it.

Reply:
This is pretty wide spread. I found it in the following, non-exhaustive list of repos:

* cardano-node
* daedalus
* cardano-ledger
* cardano-rest
* cardano-explorer
* cardano-base
* cardano-sl
* cardano-db-sync

Reply:
i suspect it isn't needed, and things will still pass if its taken back out

Reply:
I suspect it is. I'd have a chat with @erikd to confirm.

IIRC we had issues with at least one build where add this line resolved it.

Reply:
I removed it because it's not required.